piff/decamutil.py

```diff
@@ -0,0 +1,214 @@
+#
+# $Rev::                                                              $:
+# $Author::                                                           $:
+# $LastChangedDate::                                                  $:
+#
+# Utility methods for DECam
+#
+
+import numpy as np
+
+class decaminfo(object):
+    """ decaminfo is a class used to contain DECam geometry information and various utility routines

```

Comment:
In GalSim, we have survey-specific modules in sub-directories of the main `galsim` module.  Perhaps that would be a good model here as well.  So this would be `piff.des.decaminfo`  and the wavefront would be `piff.des.DECamWavefront`.

Also, as a style point, I'd recommend `DECamInfo` for this class, rather than `decaminfo`.